game/templates/game/rules.html

```
@@ -468,7 +540,9 @@
                 style="height: 2rem; width: auto; object-fit: contain;" alt="Checkora">
             <span style="font-family: 'Cinzel', serif; color: #f0c040; font-size: 1.5rem; font-weight: 700; letter-spacing: 0.08em; text-transform: uppercase;">Checkora</span>
         </a>
+        <button id="menuToggle" onclick="toggleSidebar()">☰ Menu</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)